pandas/core/generic.py

```diff
@@ -5909,6 +5909,97 @@ def infer_objects(self: FrameOrSeries) -> FrameOrSeries:
             )
         ).__finalize__(self)
 
+    def as_nullable_dtypes(
+        self: FrameOrSeries, convert_integer: bool_t = True
+    ) -> FrameOrSeries:
+        """
+        Convert columns of DataFrame or a Series to types supporting ``pd.NA``.
+
+        If the dtype is ``object``, if possible, convert to ``StringDtype``,
+        ``BooleanDtype`` or an appropriate integer extension type, otherwise leave as
+        ``object``.
+
+        If the dtype is integer", convert to an appropriate integer type.
```

Comment:
Missing / extra quotes here.

"appropriate integer type" -> "corresponding nullable integer type"?

Reply:
I made it "appropriate integer extension type" to correspond to the other places.